resources/dicts/events/new_cat/general.json

```diff
@@ -608,7 +609,8 @@
             [
                 "old_name",
                 "clancat",
-                "status:warrior"
+                "status:warrior",
```

Comment:
In the text it's mentioned "join in exchange for healing" I thought this means "I join the clan and help you out with my healing abilities"

If so, shouldn't the rank be medicine cat?

Reply:
The text also mentions that the warrior is injured.  I'd always interpreted it as being a warrior going "hey, I'm hurt, but I'll join your Clan as a loyal warrior if you help me heal"